gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/use_case/MigrateApiUseCase.java

```diff
@@ -109,7 +114,41 @@ public Output execute(Input input) {
                     new MigrationResult.Issue("Cannot migrate an API not using V4 emulation", MigrationResult.State.IMPOSSIBLE)
                 );
         }
+        List<PageEntity> pageEntities = pageService.findByApi(input.auditInfo().environmentId(), input.apiId());
+        for (PageEntity page : pageEntities) {
+            if (page.getTranslations() != null && !page.getTranslations().isEmpty()) {
```

Comment:
```suggestion
            if (isEmpty(page.getTranslations())) {
```

It’s lightweight for reading (same for 2 next if)

Reply:
fixed